sdk/communication/communication-short-codes/CHANGELOG.md

```diff
@@ -1,5 +1,4 @@
 # Release History
-
 ## 1.0.0-beta.2 (2022-03-31)
```

Comment:
What's your release strategy going to be? Considering that the package is currently in beta, it seems that there is a public preview of another API going on. If you merge this to main, you should bump the package version to 1.1.0-alpha.1. But then releasing a stable version of your beta API will be blocked.

With the exception of JS, private preview packages are supposed to be released to different feeds (see more [here](https://dev.azure.com/skype/SPOOL/_wiki/wikis/SPOOL.wiki/14211/Public-Private-Preview-Development)).

Reply:
I didn't quite get your point. We're planning to merge a few more changes before releasing a new version. Even though this package is in beta, it's being considered a private preview release. This was discussed internally, and the team agreed that it should go under the `@azure-tools` namespace so it would be seen as a non-public release, but still allowing the portal to use it.

Reply:
I got confused by the "beta" suffix. I didn't know that @azure-tools is meant for non-public releases. Thanks for explaining.